app/src/main/res/layout/profile_fragment.xml

```diff
@@ -1,14 +1,91 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/fragment_profile"
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/fragment_profile"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".profile.ProfileFragment">
 
+    <View
+        android:id="@+id/view"
+        android:layout_width="match_parent"
+        android:layout_height="25dp"
+        android:background="?attr/colorPrimary"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <View
+        android:id="@+id/action_bar_background"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:background="?attr/colorPrimary"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/view" />
+
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:layout_marginStart="20dp"
+        android:background="?attr/colorPrimary"
+        android:text="Profile"
+        android:textAlignment="textStart"
+        android:textAppearance="@style/TextAppearance.AppCompat.Display1"
+        android:textColor="@color/white"
+        android:textSize="36sp"
+        android:visibility="visible"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/view"
+        tools:text="Profile" />
+
+    <Button
+        android:id="@+id/licenses_button"

```

Comment:
Same style concept goes for buttons.  If you have a button that is just a clickable image, the code below works well on android v21 above and below well.  

```
   <style name="MpChevronFullWidthButton" parent="Widget.AppCompat.Button.Borderless">
        <item name="android:background">?attr/selectableItemBackgroundBorderless</item>
        <item name="android:textColor">@color/greenyBlue</item>
        <item name="android:textAllCaps">false</item>
        <item name="android:fontFamily">@font/lato_regular</item>
        <item name="android: textAppearance">@style/MPowerTextAppearance </item>
        <item name="android: textAlignment"> textStart </item>
        <item name="android:textSize"> 24sp </item>
        <item name="android: drawableRight">@drawable/ic_chevron_right_black_24dp </item>
    </style>

```

then your xml element would look like this:

```
<Button
        style="@style/MpChevronFullWidthButton"
        android:id="@+id/mp_chevron_button"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_gravity="center_horizontal"/>
```